doc/rdo-packaging.txt

```diff
@@ -315,6 +315,22 @@ between the two repositories:
 * CentOS 7: http://trunk.rdoproject.org/centos70/report.html
 
 
+How to add a new package to RDO master packaging
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Prepare a public git repository with the initial .spec in rpm-master branch.
```

Comment:
Here's the plan for migration to rpm-master:
* merge https://review.gerrithub.io/216921
* require distro-branch: rpm-master in rdoinfo for new packages
* move existing packages to rpm-master one by one:
  * copy f20-master to rpm-master in openstack-packages/PROJECT repo
  * add distro-branch: rpm-master in rdoinfo for PROJECT
  * change defaultbranch in .gitreview to rpm-master
  * if there are open reviews on gerrithub, rebase them to rpm-master
  * drop f20-master branch
  * announce on rdo-list that PROJECT has been migrated
* when all projects are migrated, change default distro=rpm-master in projects.ini